nixos/modules/services/games/factorio.nix

```diff
@@ -87,6 +87,18 @@ in
           a new map with default settings will be generated before starting the service.
         '';
       };
+      loadLatestSave = mkOption {
+        type = types.bool;
+        default = true;
+        description = ''
+          Load the latest savegame on startup. This overrides saveName, in that the latest
+          save will always be used even if a saved game of the given name exists. It still
+          controls the 'canonical' name of the savegame.
+
+          Set this to true to have the server automatically reload a recent autosave after
+          a crash.
```

Comment:
```suggestion
          Set this to true to have the server automatically reload a recent save or autosave after
          a restart or desync.
```

Reply:
"crash or desync". We already have this behavior for a clean restart.